broadcasting.md

```diff
@@ -247,6 +255,47 @@ npm run dev
 > **Note**  
 > To learn more about compiling your application's JavaScript assets, please consult the documentation on [Vite](/docs/{{version}}/vite).
 
+Additional supported features :
+
+*1. Modify private/presence channel capability. Default: Full capability*
+- Channel access control rights are granted for each individual user separately using `ably-capability`.
+- It defines list of access claims as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations).
+
+```php
+  // file - routes/channels.php
+  // User authentication is allowed for private/presence channel returning truthy values and denied for falsy values.
+  
+  // for private channel
+  Broadcast::channel('channel1', function ($user) {
+      return ['ably-capability' => ["subscribe", "history"]];
```

Comment:
Would it make sense to add something to demonstrate the truthy/falsy behaviour in the examples, like:
```suggestion
      if (user.isAuthenticated()) {
          return ['ably-capability' => ["subscribe", "history"]];
      } else {
          return false;
      }
```

Reply:
I don't think we need to check if a user is authenticated, 
Those routes already have an internal guard that checks if the user is authenticated.

Reply:
You can check this -> https://github.com/laravel/framework/blob/9.x/src/Illuminate/Broadcasting/BroadcastController.php
It's being used in the file for authenticating routes inside
https://github.com/laravel/framework/blob/9.x/src/Illuminate/Broadcasting/BroadcastManager.php

Reply:
I mean otherwise, it wouldn't make sense to get an invalid user as an argument in the first place.

Reply:
looks good to me, thanks